tendermint/src/block/height.rs

```diff
@@ -69,12 +69,6 @@ impl From<Height> for u64 {
     }
 }
 
-impl From<Height> for i64 {
-    fn from(height: Height) -> Self {
-        height.0 as Self
-    }
-}
```

Comment:
Did we verify this isn't used in the KMS for instance?

Reply:
Does it matter? Isn't this commit non-breaking?

Reply:
Weird, I saw this comment on a different commit: https://github.com/interchainio/tendermint-rs/pull/122/commits/67f735866d358e025c5c836cc40781e97ac97120#r368718476. 

The correct one is https://github.com/interchainio/tendermint-rs/pull/122/commits/77509865c4281e0f15c5f9be503d2318a991b271#r368718476.

So yes we should verify we can get rid of this